<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP Socket GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnLed1On">
          <property name="text">
           <string>LED 1 ON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLed1Off">
          <property name="text">
           <string>LED 1 OFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="btnLed2On">
          <property name="text">
           <string>LED 2 ON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLed2Off">
          <property name="text">
           <string>LED 2 OFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="txtIP">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnConnect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="btnBuzzerOn">
              <property name="text">
               <string>Buzzer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnLCD">
              <property name="text">
               <string>LCD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnList">
              <property name="text">
               <string>List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClient">
              <property name="text">
               <string>Client</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClose">
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txtReceive">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>-1</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txtSend">
      <property name="minimumSize">
       <size>
        <width>608</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnLed1On</tabstop>
  <tabstop>btnLed2On</tabstop>
  <tabstop>btnLed1Off</tabstop>
  <tabstop>btnLed2Off</tabstop>
  <tabstop>txtReceive</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
